CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Enhancements
 
+* Add color indication to output of `pod outdated`  
+  [iv-mexx](https://github.com/iv-mexx)
+  [#issue_number](https://github.com/CocoaPods/CocoaPods/issues/issue_number)
```

Comment:
make sure to update this. You can use `https://github.com/CocoaPods/CocoaPods/pull/7204` if there is no issue associated with this PR.

Reply:
Also include one empty line between changelog entries.

Reply:
Oh right, thanks!